src/Microsoft.ML.Data/Transforms/ColumnSelecting.cs

```diff
@@ -741,7 +741,7 @@ public void SaveAsOnnx(OnnxContext ctx)
                         continue;
 
                     var srcVariable = ctx.GetVariableName(srcCol.Name);
-                    var dstVariable = ctx.AddIntermediateVariable(dstCol.Type, dstCol.Name, true);
+                    var dstVariable = ctx.AddIntermediateVariable(dstCol.Type, dstCol.Name);
```

Comment:
I think in general, we're trying to move away from shape inferencing, since it makes it harder to debug . Is possible to declare the right shape instead?

Reply:
When true is specified in that spot, the output shape in the graph is null and we can't get its shape. When we remove true, then the correct output shape is listed. I thought that we were moving more towards having the shape listed in the graph instead of not listed, is that not true?